src/sage/schemes/elliptic_curves/hom_composite.py

```diff
@@ -194,11 +194,11 @@ def _compute_factored_isogeny_prime_power(P, l, n, split=.8, velu_sqrt_bound=Non
         sage: phis = hom_composite._compute_factored_isogeny_prime_power(P,l,n)
         sage: phis == hom_composite._compute_factored_isogeny_prime_power(P,l,n, split=0)  # long time -- about 10s
         True
-        sage: phis == hom_composite._compute_factored_isogeny_prime_power(P,l,n, split=0.1)
+        sage: phis == hom_composite._compute_factored_isogeny_prime_power(P,l,n, split=0.1)  # long time (:issue:`39569`)
         True
-        sage: phis == hom_composite._compute_factored_isogeny_prime_power(P,l,n, split=0.5)
+        sage: phis == hom_composite._compute_factored_isogeny_prime_power(P,l,n, split=0.5)  # long time (:issue:`39569`)
         True
-        sage: phis == hom_composite._compute_factored_isogeny_prime_power(P,l,n, split=0.9)
+        sage: phis == hom_composite._compute_factored_isogeny_prime_power(P,l,n, split=0.9)  # long time (:issue:`39569`)
```

Comment:
Since the current CI run says that line 203 is slow as well, it might be easier to take all of these `# long time`s and move them up to the top of the test?

Reply:
in principle, tests taking more than 1 second should be marked, but for now I just look at a particular CI run and see which of these take more than 5 seconds. Those borderline will not show up very often.

Reply:
there's a `long time` mark in the line above (split=0) but not here, which makes me suspect a regression happened.